R/RestRserveResponse.R

```diff
@@ -159,28 +159,33 @@ RestRserveResponse = R6::R6Class(
     encode = NULL,
     #------------------------------------------------
     initialize = function(body = "",
-                          content_type = 'text/plain',
-                          headers = NULL,
+                          content_type = "text/plain",
+                          headers = list_named(),
                           status_code = 200L,
                           encode = NULL) {
       if (isTRUE(getOption('RestRserve_RuntimeAsserts', TRUE))) {
```

Comment:
Seems now this block not in runtime.

Reply:
not runtime (so the 'RestRserve_RuntimeAsserts' may be misleading), but users will still call `new` method. Hence we need asserts

Reply:
Sure. I mean that we can remove `if` condition.